src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public class RightTriangle extends Figure {
+    private int base;
+    private int height;
+
+    {
+        name = "right triangle";
+    }
```

Comment:
you can add a constructor to `Figure` class, and set the name using `super()` keyword

Reply:
Can you clarify this comment?

Reply:
You mean, something like this will be correct?

public Figure(String name, Color color) {
        this.name = name;
        this.color = color.name().toLowerCase();
}

----->>>

public Square(String name, int side, Color color) {
        super(name, color);
        this.side = side;
}